### PR TITLE
[sc-8602] Display pending invitations

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.service.ts [hunk 1]
```diff
@@ -14,7 +14,18 @@ export class UsersManageService {
         switchMap(([kb, account]) => kb.getUsers(account.slug)),
       ),
     ),
-    shareReplay(),
+    shareReplay(1),
+  );
+
+  invitesKb = this._onUpdateUsers.pipe(
+    switchMap(() => this.sdk.currentKb),
```
Comment (mpellerin42) [inline apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.service.ts:21]: don't you need a `take(1)` on `currentKB` to avoir problems when switching KB?

File: libs/common/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -514,6 +514,8 @@
 	"kb.settings.user_prompts.prompt_help": "Définit la manière dont nous voulons que le système agisse lorsque nous posons une question. Il est possible de définir des règles plus spécifiques, qui peuvent même remplacer le comportement défini. Par exemple, nous pourrions avoir une invite qui oblige le système à ne donner que des résultats sous forme de listes, ou même à demander de répondre dans une langue donnée.",
 	"kb.settings.user_prompts.system": "Comportement personnalisé",
 	"kb.settings.user_prompts.system_help": "Définit les traits de personnalité généraux de l'assistant, ou la manière dont il doit se comporter lorsqu'il répond à des questions ou au cours d'une conversation. Par exemple, nous pourrions définir que le système répondra en tant que professionnel de la santé.",
+	"kb.users.already-exists": "Un utilisateur avec l'adresse e-mail <strong>{{email}}</strong> existe déjà dans cette Knowledge Box",
+	"kb.users.pending-invitation": "En attente d'invitation: expire le {{date}}",
```
Comment (mpellerin42) [inline libs/common/src/assets/i18n/fr.json:518]:
```suggestion
	"kb.users.pending-invitation": "En attente d'acceptation: expire le {{date}}",
```

**Review by mpellerin42 — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.service.ts (hunk 1 above)
Comment (operramon) [reply apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.service.ts]: Fixed

File: apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.component.html [hunk 3]
```diff
@@ -69,20 +69,29 @@ <h3>{{ 'stash.users.users_list' | translate }}</h3>
     <pa-table-cell header></pa-table-cell>
   </pa-table-header>
 
-  <pa-table-row *ngFor="let user of usersKb | async">
+  <pa-table-row *ngFor="let user of userRows | async">
```
[the remark this answers is not in the document]
Comment (operramon) [reply apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.component.html]: Fixed

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**